components/CookieBanner/CookieBannerContext.tsx

```diff
@@ -0,0 +1,85 @@
+import {
+  createContext,
+  useCallback,
+  useContext,
+  useEffect,
+  useState
+} from 'react'
+import type { ReactNode } from 'react'
+import { localItem } from '../../lib/storage'
```

Comment:
Should this file be stored in a different location to avoid creating a new folder in the project root just for it?
Not a requirement, just a thought.

Reply:
I was wondering the same, I tried to minimize the number of new files from the code taken in safe-homepage repo already, but chances are that we'll have to store other things in local or session storage in the future. So I figured it would be best in a `lib` folder than in `components/CookieBanner`.